jina/clients/websocket.py

```diff
@@ -10,25 +11,27 @@
 
 
 class WebSocketClientMixin(BaseClient, ABC):
+    """A MixIn for Websocket Client.
+
+    :meth:`send_requests()`
```

Comment:
Just curious why move this doc out of the private function?

Reply:
the docstring is not about that method. It references another method and describes its behavior. Not sure what the intent was there. But I don't think it belong in under the method

Reply:
Yes, but that method is defined within this private func, I guess that's the reason the docstring was there before (under the private func).

Reply:
Ah you're right. Somehow didn't catch that. Reverted